app/web/src/api/sdf/dal/schema.ts

```diff
@@ -97,7 +97,9 @@ export const outputSocketsAndPropsFor = (
   return opts;
 };
 
-const groupedPropsFor = (schemaVariant: SchemaVariant): GroupedOptions => {
+export const groupedPropsFor = (
```

Comment:
Now that we have filtering, we needed to break out these functions. They will use one another to reduce potential side effects though, so this should go unnoticed by all other users of `inputSocketsAndPropsFor`.